packages/contracts-bedrock/src/dispute/FaultDisputeGameN.sol

```diff
@@ -1156,10 +1161,14 @@ contract FaultDisputeGame is IFaultDisputeGame, Clone, ISemver {
         // Walk up the DAG to find a claim that commits to the same trace index as `_pos`. It is
         // guaranteed that such a claim exists.
         ClaimData storage ancestor_ = claimData[_start];
-        while (ancestor_.position.raw() != traceAncestorPosValue) {
-            ancestor_ = claimData[ancestor_.parentIndex];
+        if (traceAncestorPosValue == 0) {
```

Comment:
If `traceAncestorPosValue` should the claim be `starting` one?  Or in what situation that `traceAncestorPosValue` is zero?

Reply:
Because we offset it by 1, so it's 0 here. For consistency, I've made changes to the offset logic and let the traceAncestorPosValue be 1. https://github.com/ethstorage/optimism/pull/41/commits/74242dce1d3fa38ca34c3629e0094a1ee292727a#r1683762415